src/main/java/models/Game.java

```diff
@@ -24,16 +21,9 @@ public Game(){
         error = false;
     }
 
-
-    public void buildDeck() {
-        for(int i = 2; i < 15; i++){
-            deck.add(new Card(i,Suit.Clubs));
-            deck.add(new Card(i,Suit.Hearts));
-            deck.add(new Card(i,Suit.Diamonds));
-            deck.add(new Card(i,Suit.Spades));
-        }
-    }
-
+    //
+    // buildDeck() will need to be implemented seperately on each subclass
+    //
```

Comment:
Mihai took care of doing this in his issue. This should be left alone so we don't have merge conflicts! Could you change it back?